package.json

```diff
@@ -31,15 +31,14 @@
     "@types/jest": "^24.0.15",
     "@types/rimraf": "^2.0.2",
     "@types/webpack": "^4.4.35",
-    "@typescript-eslint/eslint-plugin": "^2.6.0",
-    "@typescript-eslint/parser": "^2.6.0",
+    "@typescript-eslint/eslint-plugin": "2.22.0",
```

Comment:
How come we are locking to this version?

Reply:
Locked the versions so we don't get the same problems we had with the import order linting.